Dockerfile

```diff
@@ -27,6 +27,7 @@ RUN yum -y update postgresql-* && \
 
 ADD container-assets/container-scripts ${CONTAINER_SCRIPTS_ROOT}
 ADD container-assets/on-start.sh ${START_HOOKS_DIR}
+ADD container-assets/pre-start.sh ${APP_DATA}/src/postgresql-pre-start/
```

Comment:
Can we make this be one of the ENV vars on line 19 for consistency? (or vice-versa, get rid of the env vars on line 19?)

Reply:
Done